helpers/analytics.ts

```diff
@@ -1,8 +1,11 @@
-import { getConfig } from "@/config/config";
-
 // https://developers.google.com/analytics/devguides/collection/gtagjs/pages
+const googleAnalyticsTarget = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS;
+
+if (!googleAnalyticsTarget) {
+  throw new Error("googleAnalyticsTarget does not exist");
```

Comment:
can we just not initialize analytics and log an error? No need to crash app :D

Reply:
in this case, we will never know that we do not have this variable so I'm not convinced 🤔